ldk/javascript/examples/self-test-loop/package.json

```diff
@@ -33,7 +33,8 @@
       "filesystem": {},
       "process": {},
       "ui": {},
-      "user": {},
+      "user": {
```

Comment:
🤔

Reply:
Whoops. This needs rebased anyway - gimme a minute to wrangle some history around.